tests/components/select/test_init.py

```diff
@@ -9,6 +13,10 @@ class MockSelectEntity(SelectEntity):
     _attr_current_option = "option_one"
     _attr_options = ["option_one", "option_two", "option_three"]
 
+    async def async_select_option(self, option: str) -> None:
```

Comment:
Why do we need to implement this here when we just call super?

Reply:
Agree, it was intended to test validation, but for that I will need to test with the `custom_component` integration. Will remove it.